reviews/models.py

```diff
@@ -36,6 +37,7 @@ class Review(models.Model):
         (METC, _('Laat opnieuw beoordelen door METC')),
         (GO_POST_HOC, _('Positief advies van FETC-GW, post-hoc')),
         (NO_GO_POST_HOC, _('Negatief advies van FETC-GW, post-hoc')),
+        (UNSUBMITTED, _('Niet verder in behandeling genomen door de FETC-GW')),
```

Comment:
Unsubmitted isn't that clear. With this definition it makes sense, but as I read the word unsubmitted in a different file before I saw this, I thought it meant 'not submitted yet'. 

Maybe change it to 'NOT_CONSIDERED'?

Reply:
How about "discontinued"? Or maybe just "removed". I feel like "not_considered" implies that it never was considered.

Coming up with a good word for this has been the hardest part of the entire branch

Reply:
> How about "discontinued"? Or maybe just "removed". I feel like "not_considered" implies that it never was considered.

I'd be fine with discontinued; it still doesn't fit perfectely, but I don't think it would be confusing. I'm not a big fan of removed, as it's a bit too generic. (In other words, it doesn't really tell what happend). 

> Coming up with a good word for this has been the hardest part of the entire branch

Naming things is always much harder than it would seem, sadly.